Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -99,6 +117,16 @@ internal struct ReflectionServiceData: Sendable {
     }
     return serializedFileDescriptorProtos
   }
+
+  internal func nameOfFileContainingSymbol(named symbolName: String) throws -> String {
```

Comment:
For "basic" failure cases it makes more sense to return an optional. An error thrown from here could be anything and it's not clear to the caller why it might thrown. If this function doesn't throw and we return `nil` instead then it's quite obvious that no file was found which contained the symbol. It should be in the service where we throw a `notFound` error.